addOns/automation/src/test/resources/org/zaproxy/addon/automation/resources/template-config.yaml

```diff
@@ -13,6 +13,11 @@ env:                                   # The environment, mandatory
     progressToStdout: true             # If set will write job progress to stdout
 
 jobs:
+  - type: activeScan-policy
+    name: activeScan-policy
+    parameters:
+      name: 
+
```

Comment:
Remove extra line?

Reply:
I'd love to .. but then it fails the `ExtentionAutomationUnitTest.shouldCreateConfigTemplateFile()` test 😦 
Anyone know why?